README.md

```diff
@@ -363,6 +364,32 @@ intArrayPublisher.send([10, 2, 2, 4, 3, 8])
 ["10", "2", "2", "4", "3", "8"]
 ```
 
+### TryMapMany
+
+Projects each element of a publisher collection into a new publisher collection form.
```

Comment:
I would just merge this documentation with the `mapMany` one mentioning that there's a try-prefixed variant from which you can throw an error and show this code example

WDYT?